test/functional/ui/multibyte_spec.lua

```diff
@@ -174,4 +174,9 @@ describe('multibyte rendering: statusline', function()
                                             |
     ]])
   end)
+
+  it('hidden group %( %) does not cause invalid unicode', function()
+    command("let &statusline = '%#StatColorHi2#%(✓%#StatColorHi2#%) Q≡'")
+    screen:snapshot_util()
```

Comment:
is this intentional? Perhaps `assert_alive()` is enough.

Reply:
just forgot to push the last push